### ISSUE TITLE
Need Implementation of `JarFile`

### ISSUE THREAD

**Issue by Harsh4902** [javapathfinder/jpf-core#483]
- to support reading a jar file, we have to implement model class and native peers for `JarFile` class. 
- This can fix issue #477

**Comment by Aditya-132** [comment]
can i work on this issue

**Comment by cyrille-artho** [comment]
Yes, we do not have active ongoing work right now; there was work in progress but it has stalled.
